starter/src/assets/script.js

```diff
@@ -14,60 +12,80 @@
    - orange.jpg by Mae Mu
    - strawberry.jpg by Allec Gomes
 */
+  
+
+/* Create an array named productsArr and add all product object literals to array */
+
+
+/* Declare an empty array named cartArr to hold the items in the cart */
+
 
-/* Declare an empty array named cart to hold the items in the cart */
+// Helper function to get product
+// Most recent product interacted with
 
-/* Create a function named addProductToCart that takes in the product productId as an argument
-  - addProductToCart should get the correct product based on the productId
-  - addProductToCart should then increase the product's quantity
+
+// Gets the product from the SKU received from the frontend
+
+
+/* Create a function named addToCart that takes in the product SKU as an argument
+  - addToCart should get the correct product based on the SKU
```

Comment:
Again, not sure that productId is a helpful change.

Reply:
I agree, my team lead had requested this change but I will relay your comment back and let them know.  Thanks!